src/models/Email.ts

```diff
@@ -26,6 +27,10 @@ export class Email {
 		}
 	}
 
+	@IsString()
+	@IsNotEmpty()
+	userId!: string;
+
```

Comment:
[nitpick] is it just github or is the spacing here a bit strange?

Reply:
looks good in the idea and have ran linters. Just looking weird here.